packages/mobile/src/screens/search-results-screen/tabs/PlaylistsTab.tsx

```diff
@@ -34,6 +35,7 @@ export const PlaylistsTab = () => {
 
   return (
     <CollectionList
+      style={{ paddingTop: spacing(3) }}
```

Comment:
I also thing it'd a bit weird we have to apply custom styles to a lot of collection lists now too. Was there something fundamentally wrong with the original styles for the list?

Reply:
Word okay makes sense thanks for clarifying the long term game plan